time stamp
int32 npoints
float64 mx
float64 my
float64 var
